### PR TITLE
Updates navigation preparing for new discovery mechanism

### PR DESCRIPTION
**Pull request by YingXue**
1. With the new device discover model, when device connects, we would only know its DCM. Moved interface names from nav to a seperate page in order to show it's retrieving status in the future. This pages now shows the DCM id, interface name and its corresponding component name
image

old nav here for comparison: 
image

2. Moved interface properties, commands and telemetry from navigation sub item style to pivot/tab style
image

3. Added friendly 'no content' message in the above pivots instead of showing a blank table to the users
image

4. small css tweaks to align app title, icon and device nav items

If need to set up a pnp device to see changes in action, use this device simulator: https://github.com/Azure/azure-iot-sdk-node/blob/digitaltwins-preview/digitaltwins/samples/device/javascript/sample_device.js
Simply run: node sample_device.js in the terminal

### CODE REVIEW

File: src/localization/locales/en.json [hunk 1]
```diff
@@ -467,13 +479,15 @@
...
-            "fetch": "Retrieve events"
+            "fetch": "Retrieve events",
+            "close": "Close"
         },
         "consumerGroups": {
             "label": "Consumer Group",
             "tooltip": "Consumer groups are used by applications to pull data from the IoT Hub. To change the current consumer group, stop monitoring telemetry."
         },
         "headerText": "Telemetry",
+        "noEvent": "Interface {{interfaceName}} contains no telemetry definition",
```
Comment (montgomp) [inline src/localization/locales/en.json]: should "definition" be plural "definitions"?

File: src/localization/locales/en.json [hunk 2]
```diff
@@ -431,10 +441,11 @@
             "expandAll": "Expand all",
             "expand": "Expand",
             "collapseAll": "Collapse all",
-            "collapse": "Collapse"
+            "collapse": "Collapse",
+            "close": "Close"
         },
         "headerText": "Commands",
-        "noCommands": "No commands",
+        "noCommands": "Interface {{interfaceName}} contains no command definition",
```
Comment (montgomp) [inline src/localization/locales/en.json]: should "definition" be plural "definitions"?

File: src/localization/locales/en.json [hunk 3]
```diff
@@ -396,10 +405,11 @@
             "expandAll": "Expand all",
             "expand": "Expand",
             "collapseAll": "Collapse all",
-            "collapse": "Collapse"
+            "collapse": "Collapse",
+            "close": "Close"
         },
         "headerText": "Writable properties",
-        "noSetting": "No settings",
+        "noSettings": "Interface {{interfaceName}} contains no writable property definition",
```
Comment (montgomp) [inline src/localization/locales/en.json]: should "definition" be plural "definitions"?

File: src/localization/locales/en.json [hunk 4]
```diff
@@ -368,13 +370,20 @@
...
     "deviceProperties": {
         "command" : {
             "refresh": "Refresh",
-            "openReportedValuePanel": "Click to view value"
+            "openReportedValuePanel": "Click to view value",
+            "close": "Close"
         },
         "headerText": "Non-writable properties",
-        "noProperty": "No properties",
+        "noProperties": "Interface {{interfaceName}} contains no non-writable property definition",
```
Comment (montgomp) [inline src/localization/locales/en.json]: should "non-writable" be "read-only"?
should "definition" be plural "definitions"?

Comment (YingXue) [reply src/localization/locales/en.json]: The non-writable is the term pnp uses in external docs, so I am keeping it this way.

**Pull request merged**